File: lesson-02/java/Program2.java
public class Program2 {

    static int count = 0;

    static int get(int n) {
        int sum = 0;

        while (n > 0) {
            sum += n; // 0(1)
            n /= 3; // 0(1)
            count++;
        }

        return sum;
    }

    public static void main(String[] args) {
        for (int i = 1; i <= 64; i++) {
            count = 0;
            get(i);
            System.out.printf("%d -> %d\n", i, count);
        }

    }
}

File: lesson-02/js/Program2.js
let count = 0;
const get = (n) => {
    let sum = 0;

    while (n > 0) {
        sum += n;
        n = Math.floor(n / 2);
        count++;
    }

    return sum;
}

count = 0;
get(100);
console.log(count);

count = 0;
get(1000);
console.log(count);

count = 0;
get(10000);
console.log(count);